Services/UserService/Controllers/UserServiceController.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Security.Cryptography.X509Certificates;
+using System.Threading.Tasks;
+using IDeleteUserUserService.Interfaces;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Razor.Language.Intermediate;
+using UserService.Commands;
+
+namespace UserService.Controllers
+{
+    [ApiController]
+
```

Comment:
Why this empty line is here?

Reply:
> Why this empty line is here?

I deleted the lines and started writing new ones, so there was an empty line

Reply:
remove blank line